orttraining/orttraining/python/training/ortmodule/_experimental/debug.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
`debug.py` is not a descriptive name. `json_config.py` is probably more accurate

ORTModule already has debug concept (and a class for it) which has the purpose of aiding debugging.

This module doesn't debug, but rather sets configuration (related to debug and other things)

Reply:
to separate future experimental, I would recommend creating a folder `json_config`, move this code to `_load_config_from_json.py` and finally exposing it to the users through a `__init__.py` with the following

`from _load_config_from_json import load_from_json`

This way, 100% implementation for json is secluded from future experimentals and the __init__ explicitly defines what is the public API